### ISSUE TITLE
caching yoga pose images

### ISSUE THREAD

**Issue by kaylynphan** [kaylynphan/rise#4]
The original intention is to download all yoga pose images during Splash screen loading so that user experience is faster and smoother.

However, this makes testing very difficult.

Before submission, make sure you move image data download from table view scroll (what is right now) to splash screen (original intention)

If possible, learn to use Realm for this.
https://github.com/realm/realm-swift
https://www.raywenderlich.com/9220-realm-tutorial-getting-started

**Comment by kaylynphan** [comment]
After doing some research, I have 2 main questions for this issue:

1) How can I speed up the image fetching?
- Dustin's idea: Create the image using SVGKit, then use AFNetworking imageView in table view cell
- [ ] try this out

2) Where do I preform the image fetching?
It looks like the image fetching, especially there is no cache data, takes quite a while. 
- Dustin's advice is to perform the image loading after the user successfully logs in. Then, I would remove the splash screen as it would not be needed.
- However, I realized that due to user login persistence, the user does not always encounter the login screen while opening up the app. This was my initial reasoning behind creating a splash screen: the user will always encounter a splash screen, regardless of whether or not they are logged in.
- Right now, while testing the app, I found that the long loading time was a hindrance. So, as of right now, images are being loaded via each cell in the gallery. This will not be part of the end product and I hope to have a better solution.
